news/_posts/2019-12-23-open-question-not-commiting-assets-anymore.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title:  "Open Question: What about not committing generated files anymore?"
+subtitle: "Let’s talk assets, we want your opinion!"
+date:   2019-12-23 08:00:00
+authors: [ PierreRambaud, PrestaShop ]
+icon: icon-chat
+tags: [1.7, development, community]
+---
+
+<div class="alert alert-info" role="alert">
+<h4><i class='icon-chat'> </i>Open question</h4>
+This post is the first of a new kind.
+We decided to open up more with you about technical questioning and decisions for the PrestaShop project.
+In these posts, we will introduce modifications or changes we want to make, in order to **discuss it with you** and make sure we make the right decision.
+
+This one is about generated files.
+Please, give us your opinion about it in the comments at the end of the post.
+</div>
+
+## Generated files?
+
+In a library context, generated files would be what users will have in the final application.
+Files in the script tags for example.
+
+These files are the concatenated, compiled and minified versions of our library files.
+These files are called “Generated files” or “assets”.
+
+In the PrestaShop context, using a tool like Webpack to compile our files and make sure we distribute the fewest files as possible would be great. This would make the installation as fast as it could be.
+But **we can’t use this kind of tools** for the following reasons :
+- Some browsers are not supporting ES6 yet
+- Our dependencies
+- The huge lines of code
+
+So we have to do it the old way.
```

Comment:
I added this sentence because I don't really now how it works today, we could say more about it I think :)